bridge-patterns/bridge-frontend
# boiler plate frontend codebase for crisis-text-line

- What does this PR do?

This PR sets up the frontend repository for the Crisis Text Line product

- What Trello task is related to this PR?
The Trello card related to this task is [here](https://trello.com/c/2zkPKUtX)

- How do I test out this PR?

Git clone this repo `cd bridge-frontend/crisis-text-line` Then run `npm install` to get all the necessary dependencies Afterwards run npm start to test out the frontend server - but nothing much to see really - just boiler plate code.

- Which end points should are ready for testing?

There are no endpoints for testing
